.github/workflows/build.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Pack
         run: yarn pack
       - name: Archive Artifacts
-        uses: actions/upload-artifact@v2
```

Comment:
@Lms24 - I bumped this due to failing CI runs (https://github.com/getsentry/sentry-wizard/actions/runs/10769670335)